container/helm/charts/openqa/Chart.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
     version: ~0.1.0
     condition: webui.enabled
   - name: postgresql
-    version: 6.5.0
-    repository: https://charts.helm.sh/stable
+    version: 15.5.38
+    repository: https://charts.bitnami.com/bitnami
```

Comment:
So we were on a *very* old version of PostgreSQL. I guess it makes sense to upgrade. I'm wondering how the migration is handled for anyone who previously used this helm chart. Probably it isn't handled at all and breaks unless one follows our migration guide where the commands can probably not copied and pasted directly for this containerized environment. Having this said, I suppose I also don't care.

This is not only upgrading the version but also the domain and thus were this comes from. I don't know enough about this ecosystem to know whether this is a trustworthy source. I guess again I don't *really* care but it would be great if that wouldn't go completely unnoticed in a PR/commit that only says it updates the version.